helm/cas-ggircs/templates/etl/deploy-load-testing-data.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "cas-ggircs.fullname" . }}-deploy-load-testing-data
+  labels:
+    release: {{ .Release.Name }}
+{{ include "cas-ggircs.labels" . | indent 4 }}
+spec:
+  suspend: true # This cron job is intended to be triggered manually
+  schedule: "* * * * *"
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      activeDeadlineSeconds: 3600
+      template:
+        metadata:
+          labels:
+            name: {{ template "cas-ggircs.fullname" . }}-deploy-load-testing-data
+            release: {{ .Release.Name }}
+{{ include "cas-ggircs.labels" . | indent 12 }}
+        spec:
```

```suggestion
        spec:
          activeDeadlineSeconds: 3600
```

The deadline needs to be on the pod so that its resources count against the right quota. I think that you can remove the deadline in the job spec above. Also 1h is probably way too long. I'd say 10min would be more than enough, no?